frame/lending/src/lib.rs

```diff
@@ -775,8 +776,18 @@ pub mod pallet {
 				borrowers.len() <= T::MaxLiquidationBatchSize::get() as usize,
 				Error::<T>::MaxLiquidationBatchSizeExceeded
 			);
-			Self::liquidate_internal(&sender, &market_id, borrowers.clone())?;
-			Self::deposit_event(Event::LiquidationInitiated { market_id, borrowers });
+			// Unwrapped because of upper check.
+			// Perhaps borrowers ids should be in the bounded vector.
+			let borrowers =
+				BoundedVec::<_, T::MaxLiquidationBatchSize>::try_from(borrowers).unwrap();
+			let subjected_borrowers = Self::liquidate_internal(&sender, &market_id, borrowers);
+			// if at least one borrower was affected then liquidation been initiated
+			if subjected_borrowers.len() != 0 {
```

Comment:
You may use BiBoundedVec and avoid this check.

Reply:
BoundedVec used here because of liquidate_internal() function. It uses BoundedVec to avoid explicit MaxLiquidationBatchSize checking. Using BiBoundedVec in the liquidate_interanl() looks restrictive since potential user of this function may want to pass empty vector of borrowers there (for some reason). Perhaps we can use BoundedVec in dispatchable liquidate() function too.